util/buildRelease/smokeTest

```diff
@@ -9,6 +9,68 @@ set -e
 
 CWD=$(cd $(dirname $0) ; pwd)
 
+MASON=0
+DOCS=0
+HELLO=0
+MISC=0
+QUICKSTART=0
+TARGETS_SET=0
+
+# Set the above options according to the arguments passed
+while [[ $# -gt 0 ]]; do
+    key="$1"
+    case $key in
+        mason)
+            # run make mason
+            MASON=1
+            TARGETS_SET=1
+            shift
+            ;;
+        docs)
+            # run make docs
+            DOCS=1
+            TARGETS_SET=1
+            shift
+            ;;
+        hello)
+            # run make check
+            HELLO=1
+            TARGETS_SET=1
+            shift
+            ;;
+        misc)
+            # run scripts to check various properties
+            MISC=1
+            TARGETS_SET=1
+            shift
+            ;;
+        quickstart)
+            QUICKSTART=1
+            shift
+            ;;
+        *)
+            echo "Usage: smokeTest [mason] [docs] [hello] [misc] [quickstart]"
+            exit 1
+            ;;
+        esac
+done
+
+# if no targets are set, check mason, docs, and make check
+# but respect the environment variables
+# CHPL_SMOKE_SKIP_MAKE_MASON CHPL_SMOKE_SKIP_DOC CHPL_SMOKE_SKIP_MAKE_CHECK
+if [ $TARGETS_SET -eq 0 ]; then
+    if [ "${CHPL_SMOKE_SKIP_MAKE_MASON}" != "true" ] ; then
+        MASON=1
+    fi
+    if [ "${CHPL_SMOKE_SKIP_DOC}" != "true" ] ; then
+        DOCS=1
+    fi
+    if [ "${CHPL_SMOKE_SKIP_MAKE_CHECK}" != "true" ] ; then
+        HELLO=1
+    fi
+    MISC=1
+fi
+
```

Comment:
Can you print out what options are enabled here. Something like:

```sh
echo "Running with MASON=$MASON DOCS=$DOCS HELLO=$HELLO MISC=$MISC QUICKSTART=$QUICKSTART"
```

Reply:
Sure